modules/sql/src/main/scala/EnumRefs.scala

```diff
@@ -7,4 +7,5 @@ package gem.sql
  *  enums refering to others like Gpi Observing Mode. */
 object EnumRefs {
   final case class EnumRef[T <: Symbol](tag: String)
+  final case class LazyEnumRef[T <: Symbol](tag: String)
```

Comment:
Maybe we should call this `SelfEnumRef` to make it clear when to use it?

Reply:
I wasn't totally sure to use literal types here but I ended up liking that solution. WDYT?

Reply:
It would also be an issue with mutual references like if `Foo.X` refers to `Bar.Y` and vice-versa, so I think "lazy" expresses what we need.

Reply:
👍